dev/com.ibm.ws.jdbc_fat_derby/build.gradle

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+addRequiredLibraries.dependsOn addDerby
+
+buildscript {
```

Comment:
FAT buckets no longer need to have this in their build.gradle, so lines 14-29 can be removed